core/primitives/sector_file/sector_file.hpp

```diff
@@ -18,7 +20,21 @@ namespace fc::primitives::sector_file {
     FTCache = 4,
   };
 
+  const std::vector<SectorFileType> kSectorFileTypes = {
+      SectorFileType::FTUnsealed,
+      SectorFileType::FTSealed,
+      SectorFileType::FTCache};
+
+  // 10x overheads
+  const std::unordered_map<SectorFileType, int> kOverheadSeal{
+      {SectorFileType::FTUnsealed, 10},
+      {SectorFileType::FTSealed, 10},
+      {SectorFileType::FTCache, 70}  // TODO(artyom-yurin): confirm for 32G
```

Comment:
Please create a task in jira and leave the id here.

Reply:
Added